File: Project1/ClientApp/src/app/model/rezerwacjainfo.ts
import { Injectable } from "@angular/core";
import { Rezerwacja } from "../model/rezerwacja";
import { Uzytkownik } from "../model/uzytkownik";

@Injectable()
export class RezerwacjaInfo {
    public rezerwacja: Rezerwacja;
    public uzytkownik: Uzytkownik;

}

File: Project1/Models/RezerwacjaInfo.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Project1.Models
{
    public class RezerwacjaInfo
    {
        public Rezerwacja rezerwacja { get; set; }
        public Uzytkownik uzytkownik { get; set; }

        public RezerwacjaInfo( Rezerwacja _rezerwacja, Uzytkownik _uzytkownik)
        {
            rezerwacja = _rezerwacja;
            uzytkownik = _uzytkownik;
        }

    }
}